kubetest2/pkg/app/cmd.go

```diff
@@ -119,9 +124,33 @@ func runE(
 	return RealMain(opts, deployer, tester)
 }
 
+// the default is $ARTIFACTS if set, otherwise ./_artifacts
```

Comment:
this is just a nicer default for local usage, and mimics kubetest in this regard